components/src/dynamo/trtllm/tests/utils/test_response_utils.py

```
@@ -0,0 +1,193 @@
+
+        @dataclass
+        class LogProbNoToken:
+            logprob: float
+            rank: int = 1
+
+        token_logprobs = [{100: LogProbNoToken(logprob=-1.5)}]
+        output = MockCompletionOutput(
+            logprobs=token_logprobs,
+            token_ids=[100],
+        )
+        log_probs, top_logprobs = ResponseUtils.extract_logprobs(output, 0)
```

Ruff (0.14.14)

Unpacked variable `log_probs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)